<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACBrowserControlsCompleteDockWidgetQt</class>
 <widget class="QDockWidget" name="ACBrowserControlsCompleteDockWidgetQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>2000</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>250</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetClosable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Browser Controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>2000</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>250</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>455</width>
      <height>418</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBoxSimilarity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>64</height>
        </size>
       </property>
       <property name="title">
        <string>Similarity</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QListWidget" name="featuresListWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>230</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>230</width>
          <height>1000</height>
         </size>
        </property>
        <property name="toolTip">
         <string>when loading media files or pre-computed features (acl files), the available features will appear here</string>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::ListMode</enum>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxClusters">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>115</height>
        </size>
       </property>
       <property name="title">
        <string>Clusters Mode</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="labelClustersNumber">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>54</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Number</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxClustersMethod">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>161</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>KMeans (default)</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="labelClustersMethod">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>54</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Clustering</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelClustersPositions">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>54</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Positions</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxClustersPositions">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>161</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Propeller (default)</string>
         </property>
        </item>
       </widget>
       <widget class="QSpinBox" name="spinBoxClusters">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>41</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
       <widget class="QSlider" name="sliderClusters">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <zorder>comboBoxClustersMethod</zorder>
       <zorder>labelClustersMethod</zorder>
       <zorder>labelClustersPositions</zorder>
       <zorder>comboBoxClustersPositions</zorder>
       <zorder>spinBoxClusters</zorder>
       <zorder>sliderClusters</zorder>
       <zorder>labelClustersNumber</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxNeighbors">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>75</height>
        </size>
       </property>
       <property name="title">
        <string>Neighbors Mode</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QLabel" name="labelNeighborsMethod">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>54</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Neighbors</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelNeighborsPositions">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>54</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Positions:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxNeighborsPositions">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>161</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>None available</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBoxNeighborsMethod">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>161</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>None available</string>
         </property>
        </item>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetModes" native="true">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>32</height>
        </size>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayoutModes">
         <item>
          <widget class="QLabel" name="labelChosenMode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>16</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Chosen mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonClusters">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>16</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Clusters</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonNeighbors">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>16</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Neighbors</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetNavigation" native="true">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>32</height>
        </size>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>4</y>
          <width>251</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayoutNavigation">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonBack">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRecenter">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Recenter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonForward">
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
